### ISSUE TITLE
Live Reloading Not Working

### ISSUE THREAD

**Issue by Dosx001** [yusukebe/gh-markdown-preview#5]
Both ```gh markdown-preview --reload``` ```gh markdown-preview -r``` are not working.

I'm using Ubuntu with WSL, it wasn't working on Firefox on Windows. I opened the Linux version but that didn't work either. I also tried Edge and nothing.

I'm editing my README.md inside Vim. I don't expect live changes but I expected changes everytime I save changes.

**Comment by yusukebe** [comment]
Hey @Dosx001 !

I'd fixed this issue on latest release. Please upgrade `gh-markdown-preview` with these commands:

```
gh extension upgrade markdown-preview
```

Then , live reloading will work!

**Comment by Dosx001** [comment]
@yusukebe thanks for looking at my issue. Unfortunately, the update didn't fix my issue. Like I said don't care about live changes but maybe introduce a new option like ```gh markdown-preview --save```. markdown-preview will refresh the page every time you save changes.

**Comment by yusukebe** [comment]
@Dosx001, Let me get this straight...

Anyway, I have been defined that "live reloading" with `--reload` option will refresh the page every time README.md is **saved**. Similar project like [Grip](https://github.com/joeyespo/grip), `grip` command refreshes every time the document is saved. So, there is no problem I think.

Is my understanding correct?

**Comment by yusukebe** [comment]
@Dosx001 OK, I'll check later. Please wait!

**Comment by yusukebe** [comment]
Hey @Dosx001 

I've tried `gh markdown-preview -r` with MS Edge and FireFox on macOS. There is no problem. It works well. 

I think that the program can't detect file changes on your environment. I want to fix this issue, but Its' difficult to reproduce  because I don't have Windows machine :(

**Comment by yusukebe** [comment]
@Dosx001 I'm sorry many times.

I tried on Ubuntu, then the problem live reloading is not working is occurred. It might be fixed.

**Comment by Dosx001** [comment]
@yusukebe sorry I don't understand your last comment. Is it not working on Ubuntu? 

Either way, I tried it on Arch Linux with WSL and it didn't work. So I'm positive it has something to do with WSL. I wish I could help you fix the issue but I don't know Go. 

Unfortunately, I did find a vim plugin that does work when using WSL and offers some more robust features. So I will be dropping gh-markdown-preview from my workflow.
https://github.com/iamcco/markdown-preview.nvim

But before I go I think there is another issue you should fix. #7

**Comment by yusukebe** [comment]
@Dosx001 Thank you for your comment.

> sorry I don't understand your last comment. Is it not working on Ubuntu?

Sorry for the trouble. It works on Ubuntu.

There is one thing that I want to ask you. If you run `uname -m` command on your Ubuntu and Arch Linux on WSL, what is displayed? Like this:

<img width="234" alt="スクリーンショット 2021-12-02 12 39 35" src="https://user-images.githubusercontent.com/10682/144353539-5b30c600-1d2b-42a6-b0b2-b7ad1d9eeb3b.png">

Please let me know.

**Comment by Dosx001** [comment]
@yusukebe both Arch and Ubuntu outputted ```x86_64```

**Comment by yusukebe** [comment]
@Dosx001 Oh I see. Thank you!

**Comment by bulletmark** [comment]
I'm on current Arch Linux with the current version of `gh` and this extension. Live reloading does not work. Is there something I am missing, or should this issue be reopened?

**Comment by yusukebe** [comment]
Hi @bulletmark 

It seems to be a bug. We should reopen this issue.
I'll try to reproduce this issue to fix it.

Thank you!

**Comment by yusukebe** [comment]
I ran this command on my Ubuntu. But, live reloading works well. I can't reproduce this issue.

![SS](https://user-images.githubusercontent.com/10682/162408934-4eb6b25f-af27-4a9f-80c3-271e8fbbc8d3.png)

Can someone show me the screenshot like above in case of live reloading not working?

**Comment by bulletmark** [comment]
I run:
```
$ gh markdown-preview 
2022/04/08 19:57:20 Accepting connections at http://*:3333/
2022/04/08 19:57:20 Open http://*:3333/ on your browser
2022/04/08 19:57:20 GET [200] /
2022/04/08 19:57:21 Watching README.md for changes
2022/04/08 19:57:21 GET [404] /favicon.ico
2022/04/08 19:57:21 GET [200] /__/md
```

Then it just sits there and never logs anything more (or updates the browser) regardless of how many times I change README.md.

Note, I have not looked at your code but when monitoring for file changes you should not monitor the actual file, rather you should monitor the parent directory and look for changes on that file name. The first approach will not catch the case where an editor changes a file by deletion + creation. The second approach does catch that. I am using vim here so I don't think that is the issue though I remember I got caught by this once on a Mac (with my own software monitor).

**Comment by yusukebe** [comment]
@bulletmark 
Thank you for your advice.

I found this issue occurs in the situation using **Vim**. Using Vim, Live Reloading is not working on my Mac.
I'm trying to fix this bug. Please give me a second.

**Comment by yusukebe** [comment]
Hi, @bulletmark 

I released `v1.2.10` for fixing this issue. Please tell me if it works right on the new version.

**Comment by bulletmark** [comment]
Yes, seems to have fixed it for me.
